Profile/EducationProfile.cs

```diff
@@ -0,0 +1,62 @@
+﻿using MarsQA_1.Helpers;

```

Comment:
Major thing. There is no setup and teardown class

Reply:
I don't think you need to explicitly call the setup and tear down class (start.cs in this case) . It would be automatically invoked everytime you run a test.